xds/internal/xdsclient/xdsresource/unmarshal_lds_test.go

```diff
@@ -34,8 +34,8 @@ import (
 	"google.golang.org/grpc/xds/internal/xdsclient/xdsresource/version"
 	"google.golang.org/protobuf/types/known/durationpb"
 
-	v1udpatypepb "github.com/cncf/udpa/go/udpa/type/v1"
-	v3cncftypepb "github.com/cncf/xds/go/xds/type/v3"
+	v1udpaudpatypepb "github.com/cncf/udpa/go/udpa/type/v1"
```

Comment:
It's weird how there's two dimesions (xds || udpa) + (xds || udpa)

Reply:
Yeah, I don't like it either. But I couldn't come up with anything better than this.